### PR TITLE
Requires interface update for tls

### PR DESCRIPTION
**Pull request by lazypower**
This adds a method to fetch tls certificates from the wire for connecting to a TLS wrapped etcd cluster

### CODE REVIEW

**Comment by mbruzek** [comment]
+1 on this interface code. I wrote much of this so of course it is good.

**Comment by lazypower** [comment]
🐌

File: requires.py [hunk 1]
```diff
@@ -31,7 +31,12 @@ def broken(self):
...
-        """
+        '''
         Get the connection string, if available, or None.
-        """
+        '''
         return self.get_remote('connection_string')
+
+    def ssl_certificates(self):
+        ''' Return a dict with the client key and certificate '''
+        return {'client_cert': self.get_remote('client_cer'),
```
Comment (lazypower) [inline requires.py]: This will be a problem

**Comment by lazypower** [comment]
@mbruzek that was for the provides.py interface :)

this is the consumer portion of that code

**Comment by mbruzek** [comment]
+1 still looks good

**Pull request merged**